packages/roosterjs-editor-types/lib/interface/ContentEditFeatureSettings.ts

```diff
@@ -226,12 +226,6 @@ export interface TextFeatureSettings {
      *      If Whole Paragraph selected, outdent paragraph
      */
     outdentWhenTabText: boolean;
-
-    /**
-     * Requires @see ExperimentalFeatures.AutoHyphen to be enabled
-     * Automatically transform -- into hyphen, if typed between two words.
-     */
-    autoHyphen: boolean;
```

Comment:
do not remove, mark as @deprecated instead, otherwise it will become a breaking change.